packages/domain-search/src/components/domain-search-controls/input.scss

```diff
@@ -0,0 +1,22 @@
+.domain-search-controls__input {
+    width: 100%;
+
+    .components-input-base {
+        border-radius: var(--domain-search-input-border-radius);
+    }
+
+    .components-input-control__container {
+        box-shadow: inset 0 0 0 1px var(--studio-gray-20);
+        background: var(--studio-white);
```

Comment:
We can't use `--studio` variables here because they're specific to Calypso. We need to use WPDS ones, or register new ones in `.domain-search`.